packages/cli/src/services/orchestration.handler.service.ts

```diff
@@ -9,12 +9,17 @@ import {
 import { handleWorkerResponseMessage } from './orchestration/handleWorkerResponseMessage';
 import { handleCommandMessage } from './orchestration/handleCommandMessage';
 import { MessageEventBus } from '../eventbus/MessageEventBus/MessageEventBus';
+import config from '@/config';
 
 @Service()
 export class OrchestrationHandlerService {
```

Comment:
does it make sense to break this service into 3 separate services (one for each instance type) ? That might help a lot against maintaining all the `if` checks, and might also help against us forgetting to put in those checks in code at times.

Reply:
good idea